axis_utils/include/axis_utils2.inc

```diff
@@ -295,48 +295,71 @@
   !!     z(k1) would be the nearest data point to 12.5 and z(k2) would
   !!     be the nearest data point to 0.0
   !! @return integer nearest_index
-
-
-
   function NEAREST_INDEX_(value, array)
+    real(kind=FMS_AU_KIND_)              , intent(in) :: value !< arbitrary data...same units as elements in "array"
```

Comment:
Will the collide with https://github.com/NOAA-GFDL/FMS/pull/1377

Reply:
yes